source/extensions/filters/http/cache/config.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "envoy/config/filter/http/cache/v2/cache.pb.h"
+#include "envoy/config/filter/http/cache/v2/cache.pb.validate.h"
+
+#include "extensions/filters/http/cache/cache_filter.h"
+#include "extensions/filters/http/common/factory_base.h"
+#include "extensions/filters/http/well_known_names.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+
+class CacheFilterFactory
+    : public Common::FactoryBase<envoy::config::filter::http::cache::v2::CacheConfig> {
```

Comment:
This all needs to get upgraded to the v3 protos internally, either manually or via the API booster: https://github.com/envoyproxy/envoy/tree/master/tools/api_boost

Reply:
done

Reply:
I think I need to wait until I do another merge, or that I misread which places you wanted my to move to v3, or both, because going to v3 yields errors.

Reply:
AFAICT, upgrading this to v3 breaks tools/type_whisperer/typedb_gen.py, which complains that envoy.extensions.filters.http.cache.v3 "is not upgradable".

Reply:
OK not sure what is going on here. I think @htuch is out this week so if this isn't burning urgent we should ask him next week before merging.